tests/client_test.py

```diff
@@ -398,3 +409,37 @@ def test_structured_text_field(self):
                 expected_entry_occurrances -= 1
                 embedded_entry_index += 1
         self.assertEqual(expected_entry_occurrances, 0)
+
+    @vcr.use_cassette('fixtures/fields/structured_text_lists_with_embeds.yaml')
+    def test_structured_text_field_with_embeds_in_lists(self):
+        client = Client(
+            'jd7yc4wnatx3',
+            '6256b8ef7d66805ca41f2728271daf27e8fa6055873b802a813941a0fe696248',
```

Comment:
Do we want to expose the test space? I have put my credentials into env-vars, so I don't have to expose the space....

Reply:
This is CDA, and it is on by-invite alpha, I consider my test spaces for CDA should be available if anyone wants to reproduce the tests.